cli/react18/package.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Why is this in the diff?

Reply:
No clue, should be .gitignored. Will update!

Reply:
Addressed in 82b609cf5abab8e111ad1f4ce592bb0be79cc4cd